mrmustard/lab/representations/wavefunctionq.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from mrmustard.typing import Matrix, Vector
+from mrmustard.lab.representations.representation import Representation
+from mrmustard.lab.representations.data.wavefunctionarray_data import WavefunctionArrayData
+
+
+class WaveFunctionQ(Representation):
```

Comment:
How is `WaveFunctionQ` different from `WaveFunctionP` or the wavefunction in any other quadrature?